sigar_darwin.go

```diff
@@ -192,6 +192,10 @@ func (self *FileSystemList) Get() error {
 	return err
 }
 
+func (self *DiskList) Get() error {
+	return notImplemented()
```

Comment:
I know there's some historical precedent here, but this doesn't seem like the right way to be handling this.  Is there a reason not to `return errors.New("Not Implemented")`?